From 11e9d81c1cce736efcbcd8439aaad1fe4db3da3b Mon Sep 17 00:00:00 2001
From: Mikhail Tsukerman <mikhail.tsukerman@advantech.de>
Date: Tue, 4 Nov 2025 11:27:47 +0100
Subject: [PATCH] Fix imx-image-full postinstall step

Added mkdir call to create directory for /lib/firmware/nxp/sdsd8997_combo_v4.bin
---
 meta-fsl-imx/recipes-fsl/images/imx-image-full.bbappend | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meta-fsl-imx/recipes-fsl/images/imx-image-full.bbappend b/meta-fsl-imx/recipes-fsl/images/imx-image-full.bbappend
index 2e4f033..34976f8 100755
--- a/meta-fsl-imx/recipes-fsl/images/imx-image-full.bbappend
+++ b/meta-fsl-imx/recipes-fsl/images/imx-image-full.bbappend
@@ -19,6 +19,7 @@ install_utils() {
 	install -m 0755 ${ADDON_FILES_DIR}/mlanutl ${IMAGE_ROOTFS}/usr/local/bin
 	mkdir -p ${IMAGE_ROOTFS}/lib/firmware/rtlwifi/rtl8821ae
 	install -m 0755 ${ADDON_FILES_DIR}/wifi_ant_isolation.txt ${IMAGE_ROOTFS}/lib/firmware/rtlwifi/rtl8821ae
+	mkdir -p ${IMAGE_ROOTFS}/lib/firmware/nxp/
 	install -m 0644 ${ADDON_FILES_DIR}/sdsd8997_combo_v4.bin ${IMAGE_ROOTFS}/lib/firmware/nxp/sdsd8997_combo_v4.bin
 	mkdir -p ${IMAGE_ROOTFS}/lib/firmware/qca
 	install -m 0644 ${ADDON_FILES_DIR}/nvm_usb_00000302.bin ${IMAGE_ROOTFS}/lib/firmware/qca/nvm_usb_00000302.bin
-- 
2.43.0

